src/components/navigation/Link.js

```diff
@@ -0,0 +1,56 @@
+import classnames from 'classnames';
+
+import { downcastRef } from '../../util/typing';
+
+import LinkBase from './LinkBase';
+
+/**
+ * @typedef {import('../../types').PresentationalProps} CommonProps
+ * @typedef {import('preact').JSX.HTMLAttributes<HTMLAnchorElement>} HTMLAnchorAttributes
+ *
+ * @typedef LinkProps
+ * @prop {'always'|'hover'|'none'} [underline]
+ * @prop {'brand'|'text-light'|'text'} [color='brand']
+ */
+
+/**
+ * Styled component for a link (`<a>` element).
+ *
+ * @param {CommonProps & LinkProps & HTMLAnchorAttributes} props
+ */
+export default function Link({
+  children,
+  classes,
+  elementRef,
+
+  underline = 'none',
+  color = 'brand',
+
+  ...htmlAttributes
+}) {
+  return (
+    <LinkBase
+      {...htmlAttributes}
```

Comment:
Note that HTML attributes are applied in components consistently before the attributes that the component controls. In the past we've been inconsistent about this.